lein-template/src/leiningen/new/formatting.clj

```diff
@@ -0,0 +1,28 @@
+(ns leiningen.new.formatting
+  (:require [clojure.pprint :refer [pprint]]
+            [clojure.string])
+  (:import [java.io StringWriter]))
+
+(defn- pprint-to-str [v]
```

Comment:
This function can be removed and replaced with `(with-out-str (pprint v))`

Reply:
Here we also need to use the `trim-newline` function. But yes, good pointer anyway.